runtime/integration-tests/src/generic/cases/liquidity_pools.rs

```diff
@@ -7972,7 +7926,7 @@ mod centrifuge {
 					pallet_balances::Pallet::<FudgeRelayRuntime<T>>::free_balance(
 						&Keyring::Alice.into()
 					),
-					79628418552
+					79978937205
```

Comment:
quite significant changes of balances here

Reply:
Related to the above. I think it's micro CFG order.